### PR TITLE
[Ryujinx.Headless.SDL2] Address dotnet-format issues

### PR DESCRIPTION
**Pull request by TSRBerry**
Base PR: #4670

### CODE REVIEW

File: src/Ryujinx.Headless.SDL2/Program.cs [hunk 1]
```diff
@@ -195,6 +195,7 @@ private static InputConfig HandlePlayerConfiguration(string inputProfileName, st
 
                     config = new StandardControllerInputConfig
                     {
+#pragma warning disable IDE0055 // Disable formatting
```
Comment (AcK77) [inline src/Ryujinx.Headless.SDL2/Program.cs:198]: Should be removed, and code have to be formatted for consistencies

File: src/Ryujinx.Headless.SDL2/Program.cs [hunk 2]
```diff
@@ -468,20 +470,12 @@ private static void SetupProgressHandler()
...
-                case ShaderCacheState shaderCacheState:
-                    label = $"Shaders : {current}/{total}";
-                    break;
-                default:
-                    throw new ArgumentException($"Unknown Progress Handler type {typeof(T)}");
-            }
-
+                LoadState => $"PTC : {current}/{total}",
+                ShaderCacheState => $"Shaders : {current}/{total}",
+                _ => throw new ArgumentException($"Unknown Progress Handler type {typeof(T)}"),
```
Comment (AcK77) [inline src/Ryujinx.Headless.SDL2/Program.cs:477]:
```suggestion
                _ => throw new ArgumentException($"Unknown Progress Handler type {typeof(T)}")
```

File: src/Ryujinx.Headless.SDL2/Program.cs [hunk 3]
```diff
@@ -537,7 +531,7 @@ private static Switch InitializeEmulationContext(WindowBase window, IRenderer re
                 renderer = new ThreadedRenderer(renderer);
             }
 
-            HLEConfiguration configuration = new HLEConfiguration(_virtualFileSystem,
+            HLEConfiguration configuration = new(_virtualFileSystem,
                                                                   _libHacHorizonManager,
                                                                   _contentManager,
                                                                   _accountManager,
```
Comment (AcK77) [inline src/Ryujinx.Headless.SDL2/Program.cs:537]: Formatting is wrong

File: src/Ryujinx.Headless.SDL2/SDL2MouseDriver.cs [hunk 4]
```diff
@@ -46,7 +51,7 @@ private static MouseButton DriverButtonToMouseButton(uint rawButton)
 
         public void UpdatePosition()
         {
-            SDL_GetMouseState(out int posX, out int posY);
+            _ = SDL_GetMouseState(out int posX, out int posY);
```
Comment (AcK77) [inline src/Ryujinx.Headless.SDL2/SDL2MouseDriver.cs:54]: There are check on all other `SDL_` method, this one should get one too.

File: src/Ryujinx.Headless.SDL2/WindowBase.cs [hunk 5]
```diff
@@ -30,7 +31,7 @@ abstract partial class WindowBase : IHostUiHandler, IDisposable
         private const SDL_WindowFlags DefaultFlags = SDL_WindowFlags.SDL_WINDOW_ALLOW_HIGHDPI | SDL_WindowFlags.SDL_WINDOW_RESIZABLE | SDL_WindowFlags.SDL_WINDOW_INPUT_FOCUS | SDL_WindowFlags.SDL_WINDOW_SHOWN;
```
Comment (AcK77) [inline src/Ryujinx.Headless.SDL2/WindowBase.cs:31]: `protected const` vars alignment is off

**Review by AcK77 — CHANGES_REQUESTED** [review]
Some errors have to be fixed too.

File: src/Ryujinx.Headless.SDL2/SDL2MouseDriver.cs (hunk 4 above)
Comment (TSRBerry) [reply src/Ryujinx.Headless.SDL2/SDL2MouseDriver.cs:54]: It returns a bitmask of the current button state, not sure if that's useful in any way here.

**Comment by TSRBerry** [comment]
Since all checks are passing, I'll mark it as ready!

**Review by marysaka — APPROVED** [review]

File: src/Ryujinx.Headless.SDL2/SDL2MouseDriver.cs (hunk 4 above)
Comment (AcK77) [reply src/Ryujinx.Headless.SDL2/SDL2MouseDriver.cs:54]: True! Sorry I haven't checked :D

File: src/Ryujinx.Headless.SDL2/WindowBase.cs [hunk 6]
```diff
@@ -334,7 +334,7 @@ public void MainLoop()
             _exitEvent.Set();
         }
 
-        private void NVStutterWorkaround()
+        private void NvStutterWorkaround()
```
Comment (AcK77) [inline src/Ryujinx.Headless.SDL2/WindowBase.cs:337]:
```suggestion
        private void NVStutterWorkaround()
```

Same with NV

**Review by AcK77 — CHANGES_REQUESTED** [review]

**Review by AcK77 — APPROVED** [review]

**Pull request merged**